yt_dlp/extractor/heise.py

```diff
@@ -23,6 +25,10 @@ class HeiseIE(InfoExtractor):
             'timestamp': 1512734959,
             'upload_date': '20171208',
             'description': 'md5:c934cbfb326c669c2bcabcbe3d3fcd20',
+            'uploader_id': '',
```

Comment:
isnt this wrong?
```suggestion
```

Reply:
the Kaltura extractor is returning an empty string for these. the tests fail without it

Reply:
better to have failing tests than wrong tests